lib/openstax/exchange/client/fake_client.rb

```diff
@@ -0,0 +1,77 @@
+require 'securerandom'
+require 'uri'
+
+module OpenStax
+  module Exchange
+    module Client
+      class FakeClient
+        include ClientInstance
```

Comment:
I also tend to do this sometimes. You could argue that having "interfaces" like this is the java way (vs duck typing for ruby).

Since they are unnecessary, I usually end up deleting my interfaces eventually though (the method call won't even look at the included module)